protocol/app/app.go

```diff
@@ -1095,6 +1101,16 @@ func New(
 	)
 	vaultModule := vaultmodule.NewAppModule(appCodec, app.VaultKeeper)
 
+	app.ListingKeeper = *listingmodulekeeper.NewKeeper(
+		appCodec,
+		keys[listingmoduletypes.StoreKey],
+		[]string{
+			lib.GovModuleAddress.String(),
+			delaymsgmoduletypes.ModuleAddress.String(),
```

Comment:
Question, why does the delay message module need to be an authority on the listing module?

Reply:
Good catch. This was copy pasta
We could add the delay msg functionality to this module's msgs but ill remove this until we explicitly need this for a reason